server/core/src/main/java/dev/slimevr/protocol/rpc/settings/RPCSettingsHandler.kt

```diff
@@ -0,0 +1,368 @@
+				hpm
+					.setToggle(
+						SkeletonConfigToggles.FORCE_ARMS_FROM_HMD,
+						toggles.forceArmsFromHmd(),
+					)
+				hpm.setToggle(SkeletonConfigToggles.FLOOR_CLIP, toggles.floorClip())
+				hpm
+					.setToggle(
+						SkeletonConfigToggles.SKATING_CORRECTION,
+						toggles.skatingCorrection(),
+					)
+				hpm.setToggle(SkeletonConfigToggles.VIVE_EMULATION, toggles.viveEmulation())
+				hpm.setToggle(SkeletonConfigToggles.TOE_SNAP, toggles.toeSnap())
+				hpm.setToggle(SkeletonConfigToggles.FOOT_PLANT, toggles.footPlant())
+				hpm.setToggle(SkeletonConfigToggles.SELF_LOCALIZATION, toggles.selfLocalization())
+			}
+
+			if (ratios != null) {
+				if (ratios.hasImputeWaistFromChestHip()) {
+					hpm
+						.setValue(
+							SkeletonConfigValues.WAIST_FROM_CHEST_HIP_AVERAGING,
+							max(0.0, ratios.imputeWaistFromChestHip().toDouble()).toFloat(),
+						)
+				}
+				if (ratios.hasImputeWaistFromChestLegs()) {
+					hpm
+						.setValue(
+							SkeletonConfigValues.WAIST_FROM_CHEST_LEGS_AVERAGING,
+							max(0.0, ratios.imputeWaistFromChestLegs().toDouble()).toFloat(),
+						)
+				}
+				if (ratios.hasImputeHipFromChestLegs()) {
+					hpm
+						.setValue(
+							SkeletonConfigValues.HIP_FROM_CHEST_LEGS_AVERAGING,
+							max(0.0, ratios.imputeHipFromChestLegs().toDouble()).toFloat(),
+						)
+				}
+				if (ratios.hasImputeHipFromWaistLegs()) {
+					hpm
+						.setValue(
+							SkeletonConfigValues.HIP_FROM_WAIST_LEGS_AVERAGING,
+							max(0.0, ratios.imputeHipFromWaistLegs().toDouble()).toFloat(),
+						)
+				}
+				if (ratios.hasInterpHipLegs()) {
+					hpm
+						.setValue(
+							SkeletonConfigValues.HIP_LEGS_AVERAGING,
+							max(0.0, ratios.interpHipLegs().toDouble()).toFloat(),
+						)
+				}
+				if (ratios.hasInterpKneeTrackerAnkle()) {
+					hpm
+						.setValue(
+							SkeletonConfigValues.KNEE_TRACKER_ANKLE_AVERAGING,
+							max(0.0, ratios.interpKneeTrackerAnkle().toDouble()).toFloat(),
+						)
+				}
+				if (ratios.hasInterpKneeAnkle()) {
+					hpm
+						.setValue(
+							SkeletonConfigValues.KNEE_ANKLE_AVERAGING,
+							max(0.0, ratios.interpKneeAnkle().toDouble()).toFloat(),
+						)
+				}
+			}
+
+			if (legTweaks != null) {
+				if (legTweaks.hasCorrectionStrength()) {
+					legTweaksConfig.correctionStrength = legTweaks.correctionStrength()
+				}
+				api.server.humanPoseManager.updateLegTweaksConfig()
+			}
+
+			hpm.saveConfig()
+		}
+
+		val autoBoneSettings = req.autoBoneSettings()
+		if (autoBoneSettings != null) {
+			val autoBoneConfig = api.server.configManager
+				.vrConfig
+				.autoBone
+
+			RPCSettingsBuilder.readAutoBoneSettings(autoBoneSettings, autoBoneConfig)
+		}
+
+		if (req.resetsSettings() != null) {
+			val resetsConfig = api.server.configManager
+				.vrConfig
+				.resetsConfig
+			val mode = ArmsResetModes
+				.fromId(max(req.resetsSettings().armsMountingResetMode().toDouble(), 0.0).toInt())
```

Comment:
😿pls use int `max` or `coerceAtLeast`
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/coerce-at-least.html
```suggestion
				.fromId(max(req.resetsSettings().armsMountingResetMode(), 0))
```